src/live-common-set-supported-currencies.js

```diff
@@ -37,4 +37,5 @@ setSupportedCurrencies([
   "bitcoin_testnet",
   "ethereum_ropsten",
   "cosmos_testnet",
+  "celo",
```

Comment:
We'll need this PR to not enable yet the coin as we will want to merge it "ahead" and activate later.

Reply:
Sure, I removed it 👍